Scripts/fastlane/Fastfile

```diff
@@ -163,8 +163,9 @@ import "./ScreenshotFastfile"
   desc "Updates a release branch for a new beta release"
   lane :new_beta_release do | options |
     ios_betabuild_prechecks(options)
-    ios_bump_version_beta()
     ios_update_metadata(options)
+    ios_lint_localizations(input_dir: 'WordPress/Resources')
+    ios_bump_version_beta()
```

Comment:
bumping the version _after_ the linting to (1) be consistent with `finalize_release` but more importantly, so that if the linting do find violations and stops the lane (waiting for us to fix the translations or at least rejecting them then re-running the lane) we don't end up with a double version bump commit

Reply:
Excellent idea